src/ai/backend/manager/models/vfolder.py

```diff
@@ -784,22 +786,39 @@ async def load_count(
         user_id: uuid.UUID = None,
         filter: str = None,
     ) -> int:
+        from .group import groups, query_user_accessible_groups
         from .user import users
 
-        j = vfolders.join(users, vfolders.c.user == users.c.uuid, isouter=True)
+        j = (
+            sa.join(
+                vfolders,
+                vfolder_permissions,
+                vfolders.c.id == vfolder_permissions.c.vfolder,
+                isouter=True,
+            )
+            .join(users, vfolders.c.user == users.c.uuid, isouter=True)
+            .join(groups, vfolders.c.group == groups.c.id, isouter=True)
+        )
```

Comment:
JOINing triple tables in a single query is a really, really bad idea. High possibilities of slow query. Two queries with no or single join will definitely be much faster when table grows much. Please refactor this query.

Reply:
I remove joins on this code since they are not necessary. thanks!